unreleased/lokiexporter-multi-tenant.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: exporter/loki
```

Comment:
```suggestion
component: lokiexporter
```

Reply:
There actually was a discussion in the past on which format folks preferred, and having the component type followed by the component's name (without the type) was chosen:

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6688

Reply:
oh, we should start enforcing that then, the changelog is almost exclusively the other method haha